http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -489,6 +490,10 @@ if [ ! -z "$JUPYTER_DOCKER_IMAGE" ] ; then
        && cp $JUPYTER_HOME/custom/edit-mode.js $JUPYTER_USER_HOME/.jupyter/custom/ \
        && mkdir -p $JUPYTER_HOME/nbconfig"
 
+  # In new jupyter images, we should update jupyter_notebook_config.py in terra-docker.
+  # This is to make it so that older images will still work after we change notebooks location to home dir
+  docker exec ${JUPYTER_SERVER_NAME} sed -i '/^# to mount there as it effectively deletes existing files on the image/,+5d' ${JUPYTER_HOME}/jupyter_notebook_config.py
```

Comment:
tried looking up what `/^#` and `/,+5d` do with `sed` but I'm still unclear - what exactly is this command doing to `jupyter_notebook_config.py`? And will this same change need to be made in terra-docker based on this [line](https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-base/jupyter_notebook_config.py#L4)?

Reply:
it gets rid of https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-base/jupyter_notebook_config.py#L34-L38

See https://github.com/DataBiosphere/terra-docker/pull/291